src/prefect/cli/build_register.py

```diff
@@ -0,0 +1,972 @@
+import functools
+import importlib
+import json
+import multiprocessing
+import os
+import sys
+import time
+import hashlib
+import traceback
+from collections import Counter, defaultdict
+from typing import Union, NamedTuple, List, Dict, Iterator, Tuple
+
+import marshmallow
+import click
+import box
+from click.exceptions import ClickException
+
+import prefect
+from prefect.utilities.storage import extract_flow_from_file
+from prefect.utilities.filesystems import read_bytes_from_path, parse_path
+from prefect.utilities.graphql import with_args, EnumValue, compress
+from prefect.storage import Local, Module
+from prefect.run_configs import UniversalRun
+
+
+FlowLike = Union[box.Box, "prefect.Flow"]
+
+
+class SimpleFlowSchema(marshmallow.Schema):
```

Comment:
Any reason we shouldn't use pydantic here, or is it not worth the new dependency?

Reply:
I want to use pydantic, but sticking with marshmallow for now. We should maybe prioritize transitioning though.